### ISSUE TITLE
Merge with Storage

### ISSUE THREAD

**Issue by annevk** [w3c/webappsec-clear-site-data#20]
It still seems to me that most of this should end up in Storage one way or another.

**Comment by mikewest** [comment]
It's a tradeoff between explaining a thing in a capsule, and putting it into a larger whole. *shrug*

Regardless of where the header definition lives, I'd be very happy for Storage to do more of the heavy lifting in terms of describing things like "local storage areas", which are... not explained in HTML. :)

**Comment by annevk** [comment]
Yeah, the idea is that HTML carves out some space in a site storage unit's box and puts it there. I haven't really looked deeply into how to best define that though.

**Comment by annevk** [comment]
See https://github.com/whatwg/storage/issues/18 for some ideas around how to organize storage. I'd appreciate input from the folks watching this repository.

**Comment by domenic** [comment]
Merging this into somewhere where it is better maintained, to keep up with changes per partitioning and match what is actually implemented across browsers, would be really good.
